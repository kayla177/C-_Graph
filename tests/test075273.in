ENTITY 2C2XDR8G name_2C2X paper
ENTITY H2FWJ9RF name_H2FW institution
ENTITY LW2FNPRT name_LW2F journal
ENTITY BOJDF144 name_BOJD journal
ENTITY DRWI83EV name_DRWI journal
ENTITY 1BKHT4PQ name_1BKH journal
ENTITY IXOFJGIE name_IXOF paper
ENTITY ULSN3KKS name_ULSN author
ENTITY 39K85IIQ name_39K8 journal
ENTITY S59IZ0D7 name_S59I journal
RELATIONSHIP ULSN3KKS published_in IXOFJGIE 2.3
RELATIONSHIP H2FWJ9RF affiliated_with 39K85IIQ 6.8
RELATIONSHIP 1BKHT4PQ cited_by 39K85IIQ 7.0
RELATIONSHIP 39K85IIQ published_in 2C2XDR8G 7.1
RELATIONSHIP ULSN3KKS collaborated_with LW2FNPRT 3.6
RELATIONSHIP S59IZ0D7 collaborated_with H2FWJ9RF 7.9
RELATIONSHIP H2FWJ9RF affiliated_with LW2FNPRT 0.7
RELATIONSHIP 39K85IIQ authored_by LW2FNPRT 3.8
RELATIONSHIP IXOFJGIE authored_by 2C2XDR8G 7.9
RELATIONSHIP 1BKHT4PQ affiliated_with DRWI83EV 4.9
RELATIONSHIP 39K85IIQ authored_by BOJDF144 1.2
RELATIONSHIP H2FWJ9RF collaborated_with DRWI83EV 1.2
RELATIONSHIP IXOFJGIE collaborated_with S59IZ0D7 3.5
RELATIONSHIP 2C2XDR8G affiliated_with DRWI83EV 7.4
RELATIONSHIP LW2FNPRT collaborated_with DRWI83EV 6.7
DELETE BOJDF144
HIGHEST
DELETE H2FWJ9RF
PATH ULSN3KKS LW2FNPRT
HIGHEST
PATH 1BKHT4PQ 2C2XDR8G
HIGHEST
DELETE 1BKHT4PQ
PATH 39K85IIQ DRWI83EV
DELETE 39K85IIQ
EXIT
