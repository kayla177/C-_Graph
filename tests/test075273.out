success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
success
LW2FNPRT IXOFJGIE 22
success
ULSN3KKS LW2FNPRT 3.6
LW2FNPRT S59IZ0D7 25.5
1BKHT4PQ 39K85IIQ 2C2XDR8G 14.1
LW2FNPRT S59IZ0D7 25.5
success
39K85IIQ 2C2XDR8G DRWI83EV 14.5
success
